test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -66,6 +71,25 @@ test.requestHooks(mockChallengePassword)(`should load select authenticator list`
 
 });
 
+test.requestHooks(mockSelectAuthenticatorForRecovery)(`should load select authenticator list for recovery password`, async t => {
+  const selectFactorPage = await setup(t);
+  await t.expect(selectFactorPage.getFormTitle()).eql('Reset your password');
+  await t.expect(selectFactorPage.getFormSubtitle()).eql('Verify with one of the following factors to reset your password.');
```

Comment:
in verify screen in m1 we call 
Verify it's you with an authenticator

Can you run this text by vince.

Reply:
🤦  I think we shall call it `authenticator`